### PR TITLE
vjs 6 annoucement

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by gkatsev** [comment]
I got @misteroneill to look over this quickly earlier. If there are changes necessary, we can always edit it later.

File: source/_drafts/Video-js-6-0-Release.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+One of the other big changes is that source selection is now asynchronous. This was necessary for middleware support and most likely won't affect users if they are waiting for the player to be ready before interacting with the player.
+
+These are written up on our [wiki][breaking-6]. We'll make sure to update it if there's anything that we missed.
+
+## Feedback Wanted
+If you are using Video.js and have comments or questions, please drop by on [slack][]. If you find a bug, please open an [issue on github][new-issue], preferably, with a [reduced test case][reduced].
+
+## 5.x Support
+We're still going to be support the Video.js 5.x release line. This will mostly be bug fixes but features will be considered on a case-by-case basis.
```
Comment (mister-ben) [inline source/_drafts/Video-js-6-0-Release.md:35]: supporting

File: source/_drafts/Video-js-6-0-Release.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Most things have not changed between 5.x and 6.x. In fact, in most of our work in plugins we maintain, the majority of the work was to use new methods and fallback to old methods because they were logging deprecation warnings. Those plugins would've continued working otherwise.
+
+However, there are definitely some changes that are breaking and would require action on your part. For example, if you require Flash, that's something that would now need to be included manually.
+
+One of the other big changes is that source selection is now asynchronous. This was necessary for middleware support and most likely won't affect users if they are waiting for the player to be ready before interacting with the player.
+
+These are written up on our [wiki][breaking-6]. We'll make sure to update it if there's anything that we missed.
+
+## Feedback Wanted
+If you are using Video.js and have comments or questions, please drop by on [slack][]. If you find a bug, please open an [issue on github][new-issue], preferably, with a [reduced test case][reduced].
```
Comment (mister-ben) [inline source/_drafts/Video-js-6-0-Release.md:32]: Slack, GitHub

Comment (mister-ben) [reply source/_drafts/Video-js-6-0-Release.md:32]: no comma after preferably

File: source/_drafts/Video-js-6-0-Release.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Once Video.js 6 is promoted to latest, it'll take over the `next` and `latest` tags on npm. The 5.x release line will then be given its own set of tags: `latest-5` and `next-5`.
+
+## Code of Condunct
+
+We strive to be open and inclusive and so we have adopted a [Code of Conduct][coc], based on [Contributor Covenant][contrib-cov] that applies to all Video.js projects.
+
+## Conclusion
+
+We are super excited for this release! Please take it for a spin from `next` tag on npm or from the [CDN][vjs-6]. And please come chat with us on [slack][].
```
Comment (mister-ben) [inline source/_drafts/Video-js-6-0-Release.md:49]: Slack

**Review by mister-ben — COMMENTED** [review]
Just a couple of minor nitpicks

**Comment by gkatsev** [comment]
thanks @mister-ben, I updated the post.

File: source/_drafts/Video-js-6-0-Release.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  - accessibility
+  - a11y
+  - plugins
+  - Flash
+  - html5
+---
+
+After months of hard work, I am proud to annouce the release of Video.js 6.0 🎉!
+
+This release is pretty exciting. It greatly [improves the accessibility][a11y-post] of the controls and components, and we are committed to making Video.js the most accessible player we can. Video.js also provides some shiny, and awesome, new features for developers in [middleware][] and [advanced plugins][]. Video.js 6.0 is also the first release where [Flash is unbundled from core][remove-flash] -- though, it is still availablle as a plugin, if necessary.
```
Comment (chemoish) [inline source/_drafts/Video-js-6-0-Release.md:18]: `availablle`

Comment (gkatsev) [reply source/_drafts/Video-js-6-0-Release.md:18]: whoops, thanks. Fixed.

**Pull request merged**